### PR TITLE
Update to non-weapon Android Tier patches

### PR DESCRIPTION
**Pull request by Tostov**
The latest update to android tiers renamed the mod in the about file. This adds that new name to the valid modnames for the patches.

Also some tweaks to the K9 robot dog and the advanced jaw, since the latter cannot be used by humanoid androids like I previously thought.

### CODE REVIEW

File: Patches/Android Tiers/Hediffs/Hediffs_Android.xml [hunk 1]
```diff
@@ -142,10 +143,10 @@
 					<capacities>
 						<li>Bite</li>
 					</capacities>
-					<power>31</power>
-					<cooldownTime>1.86</cooldownTime>
-					<armorPenetrationSharp>4.22</armorPenetrationSharp>
-					<armorPenetrationBlunt>13.5</armorPenetrationBlunt>
+					<power>44</power>
+					<cooldownTime>2.51</cooldownTime>
+					<armorPenetrationSharp>14.07</armorPenetrationSharp>
```
Comment (N7Huntsman) [inline Patches/Android Tiers/Hediffs/Hediffs_Android.xml]: What creatures are these jaws for? They're extremely strong (for comparison, the Grizzly bear has the strongest vanilla Bite with 31 damage) and the Sharp AP may be too high.

File: Patches/Android Tiers/ThingDef_Races/Races.xml [hunk 2]
```diff
@@ -407,11 +408,11 @@
 						<capacities>
 						  <li>Bite</li>
 						</capacities>
-						<power>22</power>
-						<cooldownTime>2.22</cooldownTime>
+						<power>26</power>
+						<cooldownTime>2.46</cooldownTime>
 						<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
```
Comment (N7Huntsman) [inline Patches/Android Tiers/ThingDef_Races/Races.xml]: Depending on how exactly you envision the Android dog, the bite may be a little too powerful. Depending on what you're going for, you might compare to the husky, wolf, and warg as points of reference.

**Comment by Tostov** [comment]
First note, the cybernetic jaw is the upgraded jaw for the K9.

As for the statistics, the general idea is that the jaw for the K9 isn't really so much a regular animal jaw so much as a pnumatic pair of scissors shaped into a jaw like facade. For the regular jaw, this is more suble and is mostly just a case of it being somewhat more powerful pressure wise and and having a slower, sharper attack (since you're looking at machined blades rather than naturally formed/hardened teeth).

The cybernetic jaw however, is keyed more off of the scyther's penetration blade rather than a regular attack, representing a strictly weapon mounted in the mouth rather than a convetional "jaw" of any description. The 40+ damage is largely to get it to the point where it can reliably scissor off a leg, which is reasonably what I'd expect it to be able to do in this instance. I do think it is reasonable to lower the sharp penetration value, since it's conceptually a tool that works more off of the sheer amount of pressure rather than specifically the sharpness of the cutting edge.

The key driver behind these decisions is the sheer value of the K9 unit, it is:
A) Extremely expensive value wise, and fairly expensive cost wise, it takes 15 components + 100 plasteel and then some uranium, but is notably valued at 8.5k (notably the other attack animals, the bear and warg are 445 and 700 respectively, meaning it's between 10-20x as expensive)
B) Installing the jaw requires another 40 steel, 50 plasteel and 8 components, and slaps another 2k onto the K9's cost.

File: Patches/Android Tiers/Hediffs/Hediffs_Android.xml [hunk 3]
```diff
@@ -145,7 +145,7 @@
 					</capacities>
 					<power>44</power>
 					<cooldownTime>2.51</cooldownTime>
-					<armorPenetrationSharp>14.07</armorPenetrationSharp>
+					<armorPenetrationSharp>9.38</armorPenetrationSharp>
 					<armorPenetrationBlunt>45</armorPenetrationBlunt>
```
Comment (N7Huntsman) [inline Patches/Android Tiers/Hediffs/Hediffs_Android.xml]: The new sharp value is acceptable. The blunt AP seems high for a bite, but I presume it's to get through power armor?

Comment (Tostov) [reply Patches/Android Tiers/Hediffs/Hediffs_Android.xml]: It is pretty much strictly to let it get through power armor, yeah.

**Review by N7Huntsman — APPROVED** [review]
With the adjusted stats, it works for me.

**Comment by zozilin** [comment]
I'll have to revert it. Please reopen when 1.1 is done @Tostov

**Pull request merged**